jsonschema-core.xml

```diff
@@ -873,18 +873,16 @@
                 </t>
                 <t>
                     Values for this property are defined elsewhere in this and other documents,
-                    and by other parties.  JSON Schema implementations SHOULD implement support
-                    for current and previous published drafts of JSON Schema vocabularies
-                    as deemed reasonable.
+                    and by other parties.
                 </t>
             </section>
             <section title='The "$vocabulary" Keyword'>
                 <t>
-                    The "$vocabulary" keyword, like all keywords in meta-schemas,
-                    relates to the schema described by the meta-schema, rather than
-                    to the meta-schema itself.  Specifically, "$vocabulary" declares
+                    The "$vocabulary" keyword, which appears in a meta-schema, declares
                     what sets of keywords are expected to be used in schemas described
-                    by this meta-schema.
+                    by that meta-schema, and with what semantics.  This is conceptually
```

Comment:
By saying "that meta-schema", it sounds like you're saying that `$vocabulary` will contain meta-schemas, which is not true, as it contains identifiers for vocabularies, right?
As such, I suggest replacing `that` with, `the associated`.

Reply:
@Relequestual IMO the current wording ("that meta-schema") is pretty precise and non-ambiguous. Using "associated" would be more confusing, because "appearing in" and "being associated to" are not obviously related.

Reply:
Call me picky, but the text "declares what sets of keywords" doesn't make me associate that the property names in `$vocabularies` are identifiers for vocabularies, it may make the reader expect some list of keywords or something. In this section the word "identifier" does not appear at all (just "identifying" in a cref). In contrast, in the "$schema" section it is clearly indicated that `$schema` is a "feature set identifier".

To reinforce the fact that the names of the properties in this keyword are identifiers for vocabularies, something in the likes of "The names of the object properties are used as JSON Schema vocabulary identifiers." might be stated somewhere (maybe in next paragraph after indicating that the keyword must be an object, or on a paragraph on its own just after that).

Reply:
OK I think I missread and got confused on this one.
@jgonzalezdr Please copy your above comment to a new review comment as I'm resolving this one (should you wish to).

Reply:
@Relequestual OK, I've moved it to a different review.